app/apptesting/gamm.go

```diff
@@ -51,6 +52,7 @@ func (s *KeeperTestHelper) PrepareBalancerPool() uint64 {
 	return poolId
 }
 
+// PrepareBalancerPoolWithPoolParams sets up a Balancer pool with balancer.PoolParams
```

Comment:
It's a good practice to refer to parameters by names instead of types. When someone changes a type name, they usually don't search for all methods to update comments.

However, changing a parameter name is more likely to make a developer update the method spec as well.
```suggestion
// PrepareBalancerPoolWithPoolParams sets up a Balancer pool with poolParams.
```